galaxy/export_user_files.py

```diff
@@ -57,7 +59,21 @@ def change_path( src ):
     if not os.path.exists( '/export/galaxy-central/' ):
         os.makedirs("/export/galaxy-central/")
         os.chown( "/export/galaxy-central/", int(os.environ['GALAXY_UID']), int(os.environ['GALAXY_GID']) )
+
     change_path('/galaxy-central/config/')
+
+    # copy image defaults to config/<file>.docker_sample to base derivatives on,
+    # and if there is a realized version of these files in the export directory
+    # replace Galaxy's copy with these. Use symbolic link instead of copying so
+    # deployer can update and reload Galaxy and changes will be reflected.
+    for config in [ 'galaxy.ini', 'job_conf.xml' ]:
+        image_config = os.path.join('/etc/galaxy/', config)
+        export_config = os.path.join( '/export/galaxy-central/config', config )
+        export_sample = export_config + ".docker_sample"
+        shutil.copy(image_config, export_sample)
+        if os.path.exists(export_config):
+            subprocess.call('ln -s -f %s %s' % (export_config, image_config), shell=True)
```

Comment:
Just out of interest, is you are not using `os.symlink()` because you need `-f`?